patterns/2-structured/core-team.md

```diff
@@ -0,0 +1,102 @@
+## Title
+
+Core Team
+
+## Patlet
+
+Is your InnerSource project difficult for people to work with, yet no one is interested in improving it?
+Establish a core team to take care of the project's fundamental items so that contributors are enabled to add and use the features that provide value to their scenarios.
+
+## Problem
+
+* It is difficult to contribute to the project.
+This could be due to things like:
+  * Can't run the project locally.
+  * Poor documentation.
+  * Convoluted code.
+  * Inadequate testing.
+* It is difficult to use the project.
+Some possible causes:
+  * Poor documentation (again).
+  * Frequent bugs.
+  * Unintuitive setup.
+
+## Story
+
+There's a central project that everyone depends on.
+What a great candidate for InnerSource!
+Unfortunately, the project has grown organically, with various contributions and additions slapped on haphazardly.
+Now it's an icky, black morass of code that no one understands and everyone is afraid to touch.
+It's clearly due for an overhaul, but even though everyone needs that and wants it, no one takes the time to work on it.
+
+## Context
+
+* Many teams use the project.
+* Significant tech debt.
+* Slow adoption and iteration on the project.
```

Comment:
This (slow adoption) seems to contradict (many teams use the project). Maybe what is meant here is the focus on few contributions (slow iteration)?

Reply:
Updated the other line to

 > Many teams ~use~ _need_ the project.